quiz.go

```diff
@@ -21,31 +27,45 @@ func main() {
 
 	r := csv.NewReader(bufio.NewReader(file))
 
-	for {
-		record, err := r.Read()
-		if err == io.EOF {
-			break
-		}
-		if err != nil {
-			log.Fatal(err)
-			continue
-		}
+	go func() {
```

Comment:
It doesn't super matter here, but I think a better practice would be to pass the vars this function depends on:

```.go
func(r io.Reader, correct, incorrect int) { 
// ...
}(r, numCorrect, numIncorrect)
```

If you get lazy about this, you will probably get bitten by https://github.com/golang/go/wiki/CommonMistakes at some point.

Reply:
Ah that makes tons of sense, I'll do that